packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -21,31 +21,44 @@ class ObjectInspectorView extends VMDeveloperView {
           title: 'Objects',
           icon: Icons.data_object,
         );
-  static const id = 'inspector-view';
+  static const id = 'object-inspector-view';
 
-  final programExplorerController = ProgramExplorerController();
+  @override
+  bool get showIsolateSelector => true;
 
+  @override
+  Widget build(BuildContext context) => _ObjectInspectorView();
+}
+
+class _ObjectInspectorView extends StatefulWidget {
   final controller = ObjectInspectorViewController();
 
+  final programExplorerController = ProgramExplorerController();
+
   @override
-  bool get showIsolateSelector => true;
+  State<StatefulWidget> createState() => _ObjectInspectorViewState();
 
+  State<StatefulWidget> initState() {
```

Comment:
initState is a lifecycle method on the State class (and it should return void). Move to _ObjectInspectorViewState below.